app/views/studies/new.html.erb

```diff
@@ -34,4 +36,35 @@
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<div class="modal" id="why-label-data-modal" role="dialog" aria-labelledby="covid19-contact-modal" aria-hidden="true">
+  <div class="modal-dialog modal-lg">
+    <div class="modal-content">
+      <div class="modal-header">
+        <span class="text-left" style="font-size:1.5em"><strong>Why label data?</strong></span> <button class="close" data-dismiss="modal">×</button>
+      </div>
+      <div class="modal-body">
+        <div>
+          <p>Annotating cells in your study consistently with ontology-based metadata allows your data to be accessed and used more broadly. The SCP Metadata Convention defines a set of metadata terms for consistent search across studies by leveraging established biomedical ontologies and controlled vocabulary. <strong><%= link_to 'Learn more', 'https://github.com/broadinstitute/single_cell_portal/wiki/Search-Studies', target: :_blank %></strong></p>
```

Comment:
One more nit: in "Annotating cells in your study consistently with ontology-based metadata", the word "consistently" is a slightly [dangling modifier](https://en.wikipedia.org/wiki/Dangling_modifier).  The word could be removed with no real impact (also reducing repetition with "consistent search terms" in the next sentence), i.e.:

```suggestion
          <p>Annotating cells in your study with ontology-based metadata allows your data to be accessed and used more broadly. The SCP Metadata Convention defines a set of metadata terms for consistent search across studies by leveraging established biomedical ontologies and controlled vocabulary. <strong><%= link_to 'Learn more', 'https://github.com/broadinstitute/single_cell_portal/wiki/Search-Studies', target: :_blank %></strong></p>
```

Or it could be clarified, e.g. "Consistently annotating cells...".  Or it could be left alone, and few would notice!